### DISCUSSION TITLE
css transform: translateX(-100%); in tailwind?

### DISCUSSION THREAD

**Discussion by Mic1** [tailwindlabs/tailwindcss#2596] [Help] [answered]
transform-translate-x-??? thanks in advance. Just found this ... is it transform-scale-x-100?

**Comment by adamwathan** [comment, marked as answer]
Hey! You can do negative values in Tailwind by sticking a dash in front of the class. `-translate-x-full` is what you need, see docs here: https://tailwindcss.com/docs/translate#app

**Reply by Mic1** [reply]
thank you

**Reply by lewebsimple** [reply]
What about `-scale-y-100` ?
